tests/test_constructor.py

```diff
@@ -221,3 +224,33 @@ def test_click_through(self):
                 autocomplete_section="Search Suggestions"
             )
             assert resp is True
+
+    def test_exceptions(self):
```

Comment:
Given that `pytest` is in the requirements, I'd recommend to change all the
`self.assert....`

with:
`assert ...`

Reply:
Why is it better?

Reply:
pytest rewrites the code on the fly and replaces the `assert` statements with their own code.

This way, you just need to use `assert` and the comparison operator that you like instead of having to remember which is the best `self.assertSomething` function for a given comparison.

Aside from that, when the assertion fails, it will typically show you more information about the values that were passed rather than just the result of the expression. You can see more information about that [here](https://docs.pytest.org/en/latest/assert.html#assert).